xonsh/environ.py

```diff
@@ -87,6 +87,7 @@
 )
 import xonsh.prompt.base as prompt
 
+from platform import uname
```

Comment:
Please `import platform` for amalgamation

Reply:
Oops, sorry!

Reply:
Done.